### DISCUSSION TITLE
Feature suggestions: Store in disk every interation of a promp/seed

### DISCUSSION THREAD

**Discussion by dudumaroja** [hlky/stable-diffusion-webui#1148] [Feature suggestions]
Whe you made a batch of imagens form a promp and find one thst just perfect, i whould be nice to be able to regenerate it but saving in disk every interation, from 1 to n.

Thia could be useful for generation dream animations showinf how the picrure was formed, and could. Be used to fine tune the detais as you can choose like 120 steps and find the step that looks better to you and post process it.

A example of. Animation that can be done using this method 

https://www.instagram.com/reel/Cig7kXzAKnQ/?igshid=YmMyMTA2M2Y=

**Comment by ZacSchwifty** [comment]
+
Would be great in GUI under advanced options : 
1.  preview every nth iteration 
3. save every nth iteration to file
